job/job.go

```diff
@@ -84,9 +139,24 @@ func CreateFromIP(ip net.IP) (Job, error) {
 	if err != nil {
 		return Job{}, err
 	}
+
+	// check if there is an active workflow for the device
+	hd := (*d).Hardware()
+	hwID := (*hd).HardwareID()
+	activeWorkflows, err := isHardwareHavingActiveWorkflow(hwID)
+	if err != nil {
+		return Job{}, err
+	}
+
+	if !activeWorkflows {
+		s := fmt.Sprintf("no active workflow found for hardware %s", hwID)
+		return Job{}, errors.New(s)
```

Comment:
can be combined into just `errors.Errorf`

Reply:
done